### PR TITLE
Add guest blog post: [科技探討]全民皆拍的共監視時代，你準備好了嗎？

### PR DESCRIPTION
**Pull request by cyl337**

### CODE REVIEW

File: _posts/2014-09-16-age-of-coveillance.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+##[科技探討]全民皆拍的共監視時代，你準備好了嗎？
```
Comment (hupili) [inline _posts/2014-09-16-age-of-coveillance.md:11]: Needs to add Jekyll's preamble like this [example](https://raw.githubusercontent.com/code4hk/blog/gh-pages/_posts/2014-02-23-blog-is-up.md). This includes some basic meta data

Comment (cyl337) [inline _posts/2014-09-16-age-of-coveillance.md:11]: Preamble is added.

File: _posts/2014-09-16-age-of-coveillance.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+category : Technology in society
+author: CY L.
+author_url: https://github.com/cyl337
+title: "[科技探討]全民皆拍的共監視時代，你準備好了嗎？"
+tagline: 
+tags : [code4hk, blog, sousveillance, surveillance]
+---
+
```
Comment (debuggingfuture) [inline _posts/2014-09-16-age-of-coveillance.md:10]: can we add
`code4HK 博客除了公佈社群動態消息，也邀請社群成員投搞探討科技及公民社會的關係。`

File: _posts/2014-09-16-age-of-coveillance.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+Sousveillance 能提供更多角度去平衡和抗衡 Surveillance ，不過Sousveillance 自不是民眾的專利。各國警察已陸續配備佩戴式攝錄器，香港警察也已經在部分前線警員試用[5]，據香港警務處稱，對警員的無理投訴和挑釁因此大為減少。[6] [7] [8] 
+
+###錄像一定客觀？
+
+錄像片段通常予人真實記錄、客觀可靠的感覺。但就跟傳媒的報導一樣，任何錄像的角度和取裁本身就代表一種立場。而社會心理學的研究更發現，錄像的拍攝視角對觀看者如何理解所拍攝的事件有顯著影響，即所謂camera-perspective bias 。在一個實驗中[5]，模擬警員盤問疑犯的過程分別從三個不同視角進行拍攝：警員望向疑犯、疑犯望向警員、同時拍到警員和疑犯側面的第三者視點。各個視角的錄像分別播放給三組實驗參與者並調查他們的觀感。結果發現，相對於觀看第三者視點的組別，觀看警員視點的參與者更傾向認為疑犯有罪，而觀看疑犯視點的組別更多地認為警員的盤問不當地引導作供。
+
+###錄像 = 發言權
+
+但無可可否認，錄像確是比單憑證言來得客觀可靠，甚至乎令人覺得有錄像才值得相信，相反沒有錄像就沒有發言權。掌握越多資源(技術、資金、人力)能拍攝更多更清晰錄像的人，發言就越權威。雖然攝錄科技的普及將這種差距收窄，但錄像的影響力增加的趨勢同時可能放大資源差距造成的不平等，令技術弱勢者更被邊緣化。
```
Comment (debuggingfuture) [inline _posts/2014-09-16-age-of-coveillance.md:36]: 但無可可否認 ->但無可否認

File: _posts/2014-09-16-age-of-coveillance.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+大至恐怖襲擊、小至社區安寧，Surveillance 對防止犯罪和搜證都卓有成效，以至天眼幾乎無所不在。面對國家安全的大義及犯罪率下降的確實數據，單以保障私隱為由去限制監控攝錄機的使用往往顯得無力。被監視者主動拍攝為自己發聲，甚至反過來去監視監控者，似乎是更積極有效的做法。由此推展，以拍攝去監察當權者──逆向監視(Inverse Surveillance)是Sousveillance 其中一個重要範疇。現實例子包括：
+* 2007年加拿大魁北克一場示威中，警方被揭發派遣警察假扮示威者當中製造衝突。警方起初否認，但群眾拍攝的片段清楚拍到蒙面肇事者穿著警靴手握石頭。[2]
+* 2011年尼日利亞大選，民眾以手機監察選舉舉報舞弊。[3]
+* 2014年香港港鐵列車撞死唐狗事故，乘客以手機拍攝的片段揭穿港鐵的聲明與事實不符。[4]
+
+Sousveillance 能提供更多角度去平衡和抗衡 Surveillance ，不過Sousveillance 自不是民眾的專利。各國警察已陸續配備佩戴式攝錄器，香港警察也已經在部分前線警員試用[5]，據香港警務處稱，對警員的無理投訴和挑釁因此大為減少。[6] [7] [8] 
+
+###錄像一定客觀？
+
+錄像片段通常予人真實記錄、客觀可靠的感覺。但就跟傳媒的報導一樣，任何錄像的角度和取裁本身就代表一種立場。而社會心理學的研究更發現，錄像的拍攝視角對觀看者如何理解所拍攝的事件有顯著影響，即所謂camera-perspective bias 。在一個實驗中[5]，模擬警員盤問疑犯的過程分別從三個不同視角進行拍攝：警員望向疑犯、疑犯望向警員、同時拍到警員和疑犯側面的第三者視點。各個視角的錄像分別播放給三組實驗參與者並調查他們的觀感。結果發現，相對於觀看第三者視點的組別，觀看警員視點的參與者更傾向認為疑犯有罪，而觀看疑犯視點的組別更多地認為警員的盤問不當地引導作供。
```
Comment (debuggingfuture) [inline _posts/2014-09-16-age-of-coveillance.md]: 取裁－＞取材

File: _posts/2014-09-16-age-of-coveillance.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+###錄像 = 發言權
+
+但無可可否認，錄像確是比單憑證言來得客觀可靠，甚至乎令人覺得有錄像才值得相信，相反沒有錄像就沒有發言權。掌握越多資源(技術、資金、人力)能拍攝更多更清晰錄像的人，發言就越權威。雖然攝錄科技的普及將這種差距收窄，但錄像的影響力增加的趨勢同時可能放大資源差距造成的不平等，令技術弱勢者更被邊緣化。
+
+因此，面對幾乎全民皆拍，錄像泛濫的時代，除了要進一步推動拍攝技術的普及至技術弱勢者，也必須教育大眾認識這種科技發展所帶來的各種社會轉變。
```
Comment (debuggingfuture) [inline _posts/2014-09-16-age-of-coveillance.md]: 全民皆拍，錄像泛濫
－＞全民皆拍 `、` 錄像泛濫

**Comment by debuggingfuture** [comment]
nice one! will merge once above resolved

File: _posts/2014-09-16-age-of-coveillance.md (hunk 2 above)
Comment (cyl337) [inline _posts/2014-09-16-age-of-coveillance.md:10]: Do you mean as tagline?

**Pull request merged**